src/NumericalAlgorithms/FiniteDifference/Unlimited.hpp

```diff
@@ -0,0 +1,83 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+
+#include "NumericalAlgorithms/FiniteDifference/Reconstruct.hpp"
+#include "Utilities/ForceInline.hpp"
+#include "Utilities/Gsl.hpp"
+
+/// \cond
+template <size_t Dim>
+class Direction;
+template <size_t Dim>
+class Index;
+/// \endcond
+
+namespace fd::reconstruction {
+namespace detail {
+template <size_t Degree>
+struct UnlimitedReconstructor {
+  static_assert(Degree == 2 or Degree == 4);
+  SPECTRE_ALWAYS_INLINE static std::array<double, 2> pointwise(
+      const double* const q, const int stride) {
```

Comment:
Probably so it can be negative :) I don't remember. This is way beyond the scope of this PR though. It would require a careful re-examining of the reconstruction code. While that's necessary, it's necessary for performance reasons and for unstructured meshes